/home/ferrazzi/.conda/envs/gpu_venv_lates/lib/python3.8/site-packages/huggingface_hub/utils/_runtime.py:184: UserWarning: Pydantic is installed but cannot be imported. Please check your installation. `huggingface_hub` will default to not using Pydantic. Error message: '{e}'
  warnings.warn(
Loading checkpoint shards:   0%|          | 0/4 [00:00<?, ?it/s]Loading checkpoint shards:  25%|██▌       | 1/4 [00:07<00:21,  7.08s/it]Loading checkpoint shards:  50%|█████     | 2/4 [00:13<00:13,  6.92s/it]Loading checkpoint shards:  75%|███████▌  | 3/4 [00:20<00:06,  6.90s/it]Loading checkpoint shards: 100%|██████████| 4/4 [00:23<00:00,  5.36s/it]Loading checkpoint shards: 100%|██████████| 4/4 [00:23<00:00,  5.94s/it]
You are calling `save_pretrained` to a 4-bit converted model, but your `bitsandbytes` version doesn't support it. If you want to save 4-bit models, make sure to have `bitsandbytes>=0.41.3` installed.
generating responses:   0%|          | 0/681 [00:00<?, ?it/s]generating responses:   0%|          | 0/681 [00:00<?, ?it/s]
generating responses:   0%|          | 0/681 [00:00<?, ?it/s]generating responses:   0%|          | 0/681 [00:00<?, ?it/s]
generating responses:   0%|          | 0/681 [00:00<?, ?it/s]generating responses:   0%|          | 0/681 [00:00<?, ?it/s]
generating responses:   0%|          | 0/681 [00:00<?, ?it/s]generating responses:   0%|          | 0/681 [00:00<?, ?it/s]
generating responses:   0%|          | 0/681 [00:00<?, ?it/s]generating responses:   0%|          | 0/681 [00:00<?, ?it/s]
